packages/vite-plugin-cloudflare/src/assets/asset-worker.ts

```diff
@@ -0,0 +1,44 @@
+// @ts-ignore
+import AssetWorker from '@cloudflare/workers-shared/dist/asset-worker.mjs';
+import { UNKNOWN_HOST } from '../shared';
+import type { WorkerEntrypoint } from 'cloudflare:workers';
+
+interface Env {
+	__VITE_ASSET_EXISTS__: Fetcher;
+	__VITE_FETCH_ASSET__: Fetcher;
+}
+
+export default class CustomAssetWorker extends (AssetWorker as typeof WorkerEntrypoint<Env>) {
+	override async fetch(request: Request): Promise<Response> {
+		const response = await super.fetch!(request);
+		const headers = new Headers(response.headers);
+		headers.delete('ETag');
+		headers.delete('Cache-Control');
+
+		// Do we need to do anything else here to duplicate the original response?
```

Comment:
This appears to be the official approach to doing this:

https://developers.cloudflare.com/workers/examples/alter-headers/

Reply:
Thanks. I'll change it to use that approach.

Reply:
08eb35e